backend/src/routes/analytics.py

```
@@ -0,0 +1,92 @@
+
+        # Calculate PnL for active positions (optional/simplified)
+        current_active_pnl = 0.0
+        for pos in active_positions:
+            try:
+                # This could be slow if there are many positions, but usually it's few
+                token_data = await data_fetcher.get_token_by_address(pos['token_address'])
+                if token_data:
+                    current_value_sol = (pos['amount_tokens'] * token_data['price']) / (wallet_info.get('sol_price', 1) or 1)
+                    current_active_pnl += (current_value_sol - pos['buy_amount_sol'])
+            except Exception:
+                continue
```

Ruff (0.15.6)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)